devs/docs/es-backports.rst

```diff
@@ -71,8 +71,8 @@ should be crossed out as well.
   does not affect too any components, so delaying it doesn't make applying
   other patches more difficult
 
-- [ ] 20d2313d4b6 Ensure ReplicationOperation notify listener once (#68256)
```

Comment:
It seems like `Ensure ReplicationOperation notify listener once `https://github.com/elastic/elasticsearch/commit/20d2313d4b6b1dfb3e1de5f6fb0286f12ea1c78c marked but not ported?

Reply:
We had already applied that change before. Cherry-picking the commit resulted in an empty diff.